<launch>

  <arg name="jid" default="tv@quicker.fr" />
  <arg name="password" default="tv" />
  <param name="tv_jabber_id" value="$(arg jid)"/>
  <param name="tv_jabber_password" value="$(arg password)"/>

  <node pkg="robair_app" type="client_manager.py" name="client_manager" required="true" output="screen"/>

</launch>